.github/workflows/testing.yml

```
@@ -29,6 +29,31 @@ env:
 jobs:
+  jest-tests:
+    if: ${{ (github.event_name == 'pull_request' && github.event.pull_request.draft == false) || (github.event.inputs.run_job == 'jest-tests') }}
+    name: Run Jest Tests
+    defaults:
+      run:
+        working-directory: ./dashboard/
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)